var/spack/repos/builtin/packages/openspeedshop-utils/package.py

```diff
@@ -0,0 +1,343 @@
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+##############################################################################
+# Copyright (c) 2015-2018 Krell Institute. All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the License, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+# Place, Suite 330, Boston, MA  02111-1307  USA
+##############################################################################
+
+from spack import *
+
+import spack
+import spack.store
+
+import os
+import os.path
+
+
+class OpenspeedshopUtils(CMakePackage):
+    """OpenSpeedShop is a community effort by The Krell Institute with
+       current direct funding from DOEs NNSA.  It builds on top of a
+       broad list of community infrastructures, most notably Dyninst
+       and MRNet from UW, libmonitor from Rice, and PAPI from UTK.
+       OpenSpeedShop is an open source multi platform Linux performance
+       tool which is targeted to support performance analysis of
+       applications running on both single node and large scale IA64,
+       IA32, EM64T, AMD64, PPC, ARM, Power8, Intel Phi, Blue Gene and
+       Cray platforms.  OpenSpeedShop development is hosted by the Krell
+       Institute. The infrastructure and base components of OpenSpeedShop
+       are released as open source code primarily under LGPL.
```

Comment:
This description doesn't really tell me anything about what this package is for.

Reply:
Ok.  I need to explain what openspeedshop-utils is.  I will update.